client/js/components/make-sticky.js

```diff
@@ -0,0 +1,22 @@
+import {nodeList, featureTest} from '../util';
+import debounce from 'lodash.debounce';
+
+const makeSticky = (els) => {
+  if (featureTest('position', 'sticky')) {
+    const elements = nodeList(els);
+    const applyPositioning = () => {
+      elements.map((e) => {
+        if (e.offsetHeight > window.innerHeight) {
+          e.classList.remove('sticky-applied');
+        } else {
+          e.classList.add('sticky-applied');
+        }
+      });
+    };
+
+    document.addEventListener('readystatechange', applyPositioning);
```

Comment:
Will this fire three times? Once for 'loading', 'interactive' and 'complete' states?